server/src/controllers/course.ts

```diff
@@ -17,3 +17,9 @@ export interface Course {
   name: LocalizedString,
   evaluationInformation: LocalizedString
 }
+
+export enum Language {
```

Comment:
Should we use enums 'EN', 'FI' and 'SV' to match the database language enums?

Reply:
If possible? Having arbitrary numbers as language identifiers can be confusing. Matching with db enums should make querying easier when querying for example course translations.